server/auth/types/authentication_type.ts

```diff
@@ -119,7 +119,7 @@ export abstract class AuthenticationType implements IAuthenticationType {
       try {
         const additionalAuthHeader = await this.getAdditionalAuthHeader(request);
         Object.assign(authHeaders, additionalAuthHeader);
-        authInfo = await this.securityClient.authinfo(request, additionalAuthHeader);
+        authInfo = await this.securityClient.authinfo(request, '', additionalAuthHeader);
```

Comment:
Do we need to make the addition of this the default behavior?

Reply:
yes, it is required as an argument for `authinfo` method. '' here since this block will only be called on rest-API calls only, and for rest calls made as Anonymous user, the backend will evaluate this as `opendistro_security_anonymous` since no auth header would have been passed, so '' will work.